src/GitHub.Exports/Primitives/UriString.cs

```diff
@@ -197,7 +166,7 @@ public override UriString Combine(string addition)
             if (url != null)
             {
                 var urlBuilder = new UriBuilder(url);
-                if (!String.IsNullOrEmpty(urlBuilder.Query))
+                if (!string.IsNullOrEmpty(urlBuilder.Query))
```

Comment:
Is there an advantage to doing this with the value type instead of the class type?